Odata-docs/TOC.yml

```diff
@@ -334,6 +334,8 @@
         href: /odata/odatalib/optional-parameters
       - name: Set schema for out-of-line annotations
         href: /odata/odatalib/edm/set-annotations-schema
+      - name: Use IEdmTargetPath for out-of-line annotations target
```

Comment:
Maybe "Set target for out-of-line annotations" - IEdmTargetPath interface being one of means at your disposal to do that...
You could demonstrate in this doc how to set the target for out-of-line annotations using the 2 (AFAIK) ways

Reply:
Opted to use @xuzhg proposal `Enable annotations on target path`

We will just focus on using the EdmTargetPath. The other way defining out of line annotations is defined [here](https://learn.microsoft.com/en-us/odata/odatalib/edm/define-annotations)